### ISSUE TITLE
Usability:  Transaction selection and options

### ISSUE THREAD

**Issue by MikeBishop** [actualbudget/releases#51]
Currently, clicking over any editable field of a transaction starts editing that field.  Clicking the three-dot menu while focus is in a transaction field unselects the field and shows options for the account.

The only way to select a transaction is the (very small) checkbox in the left-hand column, and the only way to select multiple transactions is to select multiple very small checkboxes.  Options for a transaction (or transactions) only appear when they're selected.

I'd suggest that:

- Focus in a transaction field also select that transaction, causing the transaction options to appear in the menu
- Ctrl-click and Shift-click be supported ways of selecting multiple transactions

**Comment by btrautmann** [comment]
Shift+click does work, but there's a bug when doing so with scheduled transactions, which I've reported here: https://github.com/actualbudget/releases/issues/54

**Comment by bpaulien** [comment]
Yes, I'm finding this highly annoying.  I want the entire transaction to be selected if I click on it.  It's fine that it starts editing whatever field you clicked on...  Just select the transaction while you're doing it.
